Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -73,41 +73,64 @@ final class AttributeTests: XCTestCase {
   func testMissingClosingParenToAttribute() {
     assertParse(
       """
-      @_specializeℹ️(e1️⃣
+      @_specializeℹ️(1️⃣e2️⃣
       """,
       diagnostics: [
         DiagnosticSpec(
+          locationMarker: "1️⃣",
+          message: "unknown parameter 'e' in attribute argument; valid parameters 'target', 'availability', 'exported', 'kind', 'spi', 'spiModule', and 'available'"
```

Comment:
Initial I tought that just telling `unknown parameter 'e' in attribute argument`. 
But we know the valid arguments, so why not help?

I was also thinking of adding a Fix-It for each valid parameter type. But it's seems a bit verbose to me. 
Let know what you think @ahoppen